backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -98,6 +98,9 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
 
   lazy val scriptEpilogue = configurationDescriptor.backendConfig.as[Option[String]]("script-epilogue").getOrElse("sync")
 
+  lazy val temporaryDirectory = configurationDescriptor.backendConfig
+    .as[Option[String]]("temporary-directory").getOrElse("""$(mktemp -d "$PWD"/tmp.XXXXXX)""")
```

Comment:
ToL I realizing I'm getting to the point of annoying belaboring this on separate PR's, but it would really be nice if this were a field on `BackendConfigurationDescriptor` rather than looking it up w/ the knowlege of a type, special key name, and the `orElse` value.

Reply:
Good points. Is there a ticket that's tracking the improvement? This PR should be linked as an example.